test/e2e/details-tabs-template.test.ts

```diff
@@ -9,6 +9,11 @@ describe('Details Tabs', () => {
       await browser.url('/details-tabs.html');
       const page = new Page(browser);
       await page.waitForPageLoaded();
+
+      // Increase window height to ensure sticky-scrollbar doesn't overlap
+      // clickable table elements in the standard viewport size
+      await page.setWindowSize({ width: 1200, height: 800 });
```

Comment:
There is a built-in window resize inside `useBrowser` setup code: https://github.com/cloudscape-design/browser-test-tools/blob/main/src/browsers/browser-creator.ts#L56

It can be configured like this: `useBrowser({width: 1200, height: 800}, () => {})`

Your code works too, but inefficient, because it resizes the window twice

Reply:
I'll update.